.github/workflows/deploy-environment.yml

```
@@ -21,13 +21,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Create an image tag based on the commit sha
-        id: image_tag
+      - name: Get github commit sha
+        id: github
         run: |
           echo "sha=$(echo ${GITHUB_SHA::7})" >>$GITHUB_OUTPUT
```

I think the nested echo is probably unnecessary:

```suggestion
          echo "sha=${GITHUB_SHA::7}" >>$GITHUB_OUTPUT
```